appveyor.yml

```diff
@@ -18,17 +18,16 @@ environment:
   innounp: '%scoop_helpers%\innounp\innounp.exe'
   matrix:
     - PowerShell: 5
-    - PowerShell: 6
+    - PowerShell: Current
```

Comment:
I'd prefer if we list the major version manually. This prevents the CI from suddenly breaking when upstream releases an incompatible version.